src/main/java/chatter/command/Command.java

```diff
@@ -0,0 +1,51 @@
+package chatter.command;
+
+import chatter.ChatterException;
+import chatter.Storage;
+import chatter.TaskList;
+import chatter.Ui;
+
+/**
+ * Represents an abstract chatter.command.Command class.
+ *
+ * @author Anthony Tamzil
+ * @version CS2103T Individual Project AY2023/24 Semester 1
+ */
+public abstract class Command {
```

Comment:
I like your usage of abstract class for the Command class to not allow it to be instantiated